KSPCommunityFixes/KSPCommunityFixes.cs

```diff
@@ -26,6 +26,9 @@ public class KSPCommunityFixes : MonoBehaviour
 
         public static long FixedUpdateCount { get; private set; }
 
+        // Frame counter that doesn't use an external call like Time.frameCount.
+        public static long frameCount;
```

Comment:
Is there any chance that using a field instead of a property here saves performance given how often this is called?

Reply:
AFAIK, simple properties are always be inlined as a read over their backing fields, so that shouldn't matter.